src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The use of the universal selector (*) for setting styles is discouraged due to its impact on performance. Try to set these properties (margin, padding, box-sizing) only for the elements that require them.